### PR TITLE
Release Notes for 2021.12.1

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Adding Release Notes for 2021.12.1
- **Is this PR related to a Monolith release?** Yes, 2021.12.1

### Resolves:

* [OKTA-452953](https://oktainc.atlassian.net/browse/OKTA-452953)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md [hunk 1]
```diff
@@ -80,10 +81,15 @@ The properties in the response are:
 * `device_code`: The long string that the device uses to exchange for an access token.
 * `user_code`: The text that you enter at the URL that is listed as the value for `verification_uri`.
 * `verification_uri`: The URL that the user needs to access from their device to start the sign-in process.
+* `verification_uri_complete`: The URL that the client uses to generate the QR Code for the user to scan.
 * `expires_in`: The number of seconds that this set of values is valid. After the device code and user code expire, the user has to start the device verification process over.
 * `interval`: The number of seconds that the device should wait between polling to see if the user has finished signing in.
 
-The `user_code` and `verification_uri` must appear on the smart device for the user.
+The `user_code` and `verification_uri` must appear on the smart device for the user. To display the QR code, the client generates the code using the `verification_uri_complete` value returned in the response and displays it on the device for the user to scan.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md:88]:
```suggestion
The `user_code` and `verification_uri` must appear on the smart device for the user. To display the QR code, the client generates the code by using the `verification_uri_complete` value that was returned in the response and displays it on the device for the user to scan.
```

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 2]
```diff
@@ -3,6 +3,17 @@ title: Okta API Products release notes 2021
...
 
+### Weekly release 2021.12.1
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Okta Sign-In Widget, version 5.14.1](#okta-sign-in-widget-version-5-14-1) | December 15, 2021 |
+
+#### Okta Sign-In Widget, version 5.14.1
+
+See the [Okta Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.14.1) for more information about this release. See the [Okta Sign-In Widget guide](/code/javascript/okta_sign-in_widget/) for more information about the Widget. <!--OKTA-452958-->
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:14]:
```suggestion
See the [Okta Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.14.1) for more information on this release. See the [Okta Sign-In Widget guide](/code/javascript/okta_sign-in_widget/) for more information on the Widget. <!--OKTA-452958-->
```

**Review by christinechoy-okta — APPROVED** [review]
Just a few nits, otherwise LGTM!

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md (hunk 2 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021/index.md:14]: I'm ok with your suggestion, but the original blurb was pre-approved in the last release. I'm going to confirm with style/rn admins before I update this.

Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021/index.md:14]: If it's originally approved, keep it as is then.

**Pull request merged**